core/commands/config.go

```diff
@@ -83,6 +83,8 @@ Set the value of the 'Datastore.Path' key:
 		switch strings.ToLower(key) {
 		case "identity", "identity.privkey":
 			return errors.New("cannot show or change private key through API")
+		case "remotepinservices":
+			return errors.New("cannot show remote pinning services, use 'ipfs pin remote service ls'")
```

Comment:
We need something like a starts with here. e.g. I can do `ipfs config RemotePinServices.MyService.Key`.